pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -65,11 +67,19 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			log.Errorf("Missing expected Some-Header, got %v", r.Header)
 			w.WriteHeader(http.StatusBadRequest)
 		}
-	}
-	if r.URL.Path != "/hello/sunnyvale" && r.URL.Path != "/" {
+	case "redirect":
+		http.Redirect(w, r, "/", http.StatusMovedPermanently)
+	case "remote-redirect":
```

Comment:
maybe add a test case for 10 exceeds? hosting a /forever-loop path and redirect it over there to achieve that effect?